db/db.go

```diff
@@ -389,3 +389,41 @@ func (d *Database) MarkDepositDataSetUploaded(vaultAddress common.Address, netwo
 	vault.LatestDepositDataSetIndex++
 	return nil
 }
+
+func (d *Database) MarkValidatorRegistered(vaultAddress common.Address, network string, data []beacon.ExtendedDepositData) error {
```

Comment:
`MarkValidatorsRegistered` since there can be more than one, or `MarkDepositDataSetRegistered` to have parity with the uploaded function above.

Reply:
updated